def test_fail_on_purpose(self):
    self.assertEqual(1, 2)